docs/intro_04_interfaces.md

```diff
@@ -12,15 +12,19 @@ You can setup a Safe at https://gnosis-safe.io in [60 seconds](https://twitter.c
 
 ## Gnosis Safe Multisig Mobile
 
-While the web interface remains the focal point, there are beta apps available on mobile:
+While the web interface remains the focal point, there are also Gnosis Safe Multisig Mobile apps.
```

Comment:
Maybe we could seel it a bit more here. 

"Gnosis Safe Multisig Mobile allows you to track your assets and transactions on mobile and to stay informed on-the-go."